smart-contracts/hardhat.config.js

```diff
@@ -67,6 +67,10 @@ if (process.env.RUN_FORK) {
     },
   }
 
+  if (process.env.RUN_FORK === '137') {
+    networks.hardhat.blockGasLimit = 1_000_000_000 // 100000000429720
```

Comment:
Hum. Does this mean we bumped into gas limits??

Reply:
Yes but on the client side when fetching the route, not in the contract. That was the issue we identified while I was still using the publicProvider from our package instead or our own rpc. The issue arises with Polygon formal mostly - which may relate to how gas is computed there